aws/resource_aws_organization.go

```diff
@@ -0,0 +1,109 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"log"
+)
+
+func resourceAwsOrganization() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsOrganizationCreate,
+		Read:   resourceAwsOrganizationRead,
+		Update: resourceAwsOrganizationUpdate,
+		Delete: resourceAwsOrganizationDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"master_account_arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"master_account_email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"master_account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"feature_set": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "ALL",
```

Comment:
Minor nitpick: we should use the SDK constant here too: `organizations.OrganizationFeatureSetAll`